web/src/services/Assertion.service.ts

```diff
@@ -17,10 +17,13 @@ const getValue = (value: string): string => {
 };
 
 const getFilters = (selectors: TSpanSelector[]) =>
-  selectors.map(
-    ({key, operator = CompareOperator.EQUALS, value}) =>
-      `${key}${OperatorService.getOperatorSymbol(operator)}${getValue(value)}`
-  );
+  selectors.map(({key, operator = CompareOperator.EQUALS, value}) => {
```

Comment:
I'm changing some of this on my next PR as I'm working on moving the assertions to the backend